ui/src/pages/backoffice/Document/DocumentDetails/DocumentDetails.js

```diff
@@ -30,15 +32,24 @@ class DocumentHeader extends Component {
       <>
         <label>Document</label> #{data.metadata.pid}{' '}
         <CopyButton text={data.metadata.pid} />
+        {(
+          <>
+            <br />
+            <label>Created by</label>{' '}
```

Comment:
I think we should avoid adding these spaces around elements and use CSS styling instead. Makes the code a lot cleaner.

Reply:
Prettier does it